ViewControllers/StockDetailsViewController.m

```diff
@@ -35,29 +35,32 @@ @implementation StockDetailsViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    NSNumberFormatter *formatter = [[NSNumberFormatter alloc] init];
```

Comment:
This method is very similar to the one used for the crypto details, so this could've been a good indicator for consolidating the crypto and stocks code to avoid code duplication but would probably be too tricky to do at this point

Reply:
Yeah just a bit. I can make an attempt next week but it will be a bit tricky with the progress I've made using my current code.